src/js/components/Text/index.d.ts

```diff
@@ -9,12 +9,14 @@ import {
   PolymorphicType,
   TextAlignType 
 } from "../../utils";
+import { ExtendType } from "../../themes/base";
 
 export interface TextProps {
   a11yTitle?: A11yTitleType;
   alignSelf?: AlignSelfType;
   as?: PolymorphicType;
   color?:  ColorType;
+  extend?: ExtendType;
```

Comment:
@dallas  why do you think we should have an `extend` prop for typing while it doesn't exist for Text? It is declared on the theme styling and I think it should be enough.

Reply:
I see your related question on top ⬆️  I'm on it!